scripts/appimage/Dockerfile

```diff
@@ -96,14 +96,23 @@ RUN cd /opt && mkdir dlang && cd dlang && \
     make && mv libd_demangle.so /usr/lib/ && \
     deactivate && cd /opt && rm -Rf dlang && mv /usr/bin/gpg{~,} && mv /usr/bin/gpg2{~,}
 
+# kdesrc-build dependencies
+RUN yum install -y libcanberra-devel && \
+    cpan YAML::Syck
+
+ARG KF5_VERSION v5.101.0-rc1
```

Comment:
why this non-stable outdated version? please use the latest stable (5.110 afair)